### DISCUSSION TITLE
Bug on group actualisarion

### DISCUSSION THREAD

**Discussion by AmiardBrice** [rustdesk/rustdesk-server-pro#445] [Q&A]
Hello,

I have a strange error on the client in the group tab:

the error is: Failed to refresh group: FormatException: Missing extension byte (at offset 165) (or using trad file: Failed to refresh group: FormatException: Missing extension byte (at offset 165)

Have you ever encountered this error.

thanks in advance

**Comment by rustdesk** [comment]
@21pages

**Comment by 21pages** [comment]
Possible reason is that there are invalid utf8 characters. Could you try the program below? And show logs starting with `========flutter` in `%appdata%/RustDesk/log/rustdesk_rCURRENT.log`

https://github.com/21pages/rustdesk/releases/download/nightly/rustdesk-1.3.2-x86_64-allowMalformed.exe

**Reply by AmiardBrice** [reply]
we "found" the bug .

In one user we have added a note in french "Compte ne se connectant pas pour affecter les machines à un groupe."
in rustdesk_rCURRENT.log the "à" was replace by "Ã".

I just update the note without any accent, and it works.

An UTF-8 encoding should be a good idea...

Thanks

**Reply by 21pages** [reply]
I'm glad it could help you. Strangely, "Compte ne se connectant pas pour affecter les machines à un groupe." can be successfully decoded as UTF-8 on my computer.

**Comment by AmiardBrice** [comment]
Hello,

We found the same type of bug if device name include an accent such as "salle réunion", we can't refresh our adress book 
"FormatException: Missing extension byte (at offset 165)"

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/12693
